src/a.xml

```diff
@@ -0,0 +1 @@
+﻿<?xml version="1.0" encoding="utf-8"?><ArrayOfanyType xmlns:i="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://schemas.microsoft.com/2003/10/Serialization/Arrays"><anyType xmlns:d1p1="http://schemas.datacontract.org/2004/07/NServiceBus.Serializers.XML.Test" i:type="d1p1:SecondSerializableMessage"><d1p1:Address>53500698-a550-4648-8d83-889a7ac79661</d1p1:Address><d1p1:Age>10</d1p1:Age><d1p1:Int>7</d1p1:Int><d1p1:Name>udi</d1p1:Name><d1p1:Risk xmlns:d2p1="http://schemas.datacontract.org/2004/07/NServiceBus.Serializers.XML"><d2p1:Accuracy>0.314</d2p1:Accuracy><d2p1:Annum>true</d2p1:Annum><d2p1:Percent>0.15</d2p1:Percent></d1p1:Risk><d1p1:Uri i:nil="true" /><d1p1:Duration>-PT1H15M27.123S</d1p1:Duration><d1p1:Id>94fd631d-f245-4dd7-a8ca-b6a35268f17f</d1p1:Id><d1p1:MoreNames><d1p1:FirstSerializableMessage><d1p1:Address>dd449bc4-6124-4ee8-ad7b-ad79c3763cd1</d1p1:Address><d1p1:Age>10</d1p1:Age><d1p1:Int>7</d1p1:Int><d1p1:Name>0</d1p1:Name><d1p1:Risk xmlns:d4p1="http://schemas.datacontract.org/2004/07/NServiceBus.Serializers.XML"><d4p1:Accuracy>0.314</d4p1:Accuracy><d4p1:Annum>true</d4p1:Annum><d4p1:Percent>0.15</d4p1:Percent></d1p1:Risk><d1p1:Uri i:nil="true" /></d1p1:FirstSerializableMessage></d1p1:MoreNames><d1p1:Names><d1p1:FirstSerializableMessage><d1p1:Address>dd449bc4-6124-4ee8-ad7b-ad79c3763cd1</d1p1:Address><d1p1:Age>10</d1p1:Age><d1p1:Int>7</d1p1:Int><d1p1:Name>0</d1p1:Name><d1p1:Risk xmlns:d4p1="http://schemas.datacontract.org/2004/07/NServiceBus.Serializers.XML"><d4p1:Accuracy>0.314</d4p1:Accuracy><d4p1:Annum>true</d4p1:Annum><d4p1:Percent>0.15</d4p1:Percent></d1p1:Risk><d1p1:Uri i:nil="true" /></d1p1:FirstSerializableMessage></d1p1:Names><d1p1:Offset xmlns:d2p1="http://schemas.datacontract.org/2004/07/System"><d2p1:DateTime>2021-02-10T15:11:49.9690935Z</d2p1:DateTime><d2p1:OffsetMinutes>60</d2p1:OffsetMinutes></d1p1:Offset><d1p1:Parent><d1p1:Address>3b1099b8-6742-43ff-b859-4d304c9bfae2</d1p1:Address><d1p1:Age>10</d1p1:Age><d1p1:Int>7</d1p1:Int><d1p1:Name>-1</d1p1:Name><d1p1:Risk xmlns:d3p1="http://schemas.datacontract.org/2004/07/NServiceBus.Serializers.XML"><d3p1:Accuracy>0.314</d3p1:Accuracy><d3p1:Annum>true</d3p1:Annum><d3p1:Percent>0.15</d3p1:Percent></d1p1:Risk><d1p1:Uri i:nil="true" /></d1p1:Parent><d1p1:Some>B</d1p1:Some><d1p1:Start>2021-02-10T16:11:49.9690935+01:00</d1p1:Start></anyType></ArrayOfanyType>
```

Comment:
@andreasohlund what's this?

Reply:
hmm, that is the output from some test that for some reason wasn't ignored, I'll remove